q2_diversity/_beta/_beta_rarefaction.py

```diff
@@ -0,0 +1,166 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import pkg_resources
+import os.path
+import functools
+
+import qiime2
+import biom
+import skbio
+import seaborn as sns
+import scipy
+from emperor import Emperor
+
+import q2templates
+import q2_feature_table
+
+from ._method import beta, beta_phylogenetic, phylogenetic_metrics
+from .._ordination import pcoa
+
+TEMPLATES = pkg_resources.resource_filename('q2_diversity', '_beta')
+
+
+def beta_rarefaction(output_dir: str, table: biom.Table, metric: str,
```

Comment:
This is primarily a holdover from the older iteration of this viz, but can you unify the quotes and indent spaces (in the templates) used? If there isn't time, don't worry about it, this just might be a nice time since these files are moving around and the diff is already gross.